### PR TITLE
Add azure tf

### PR DESCRIPTION
**Pull request by micson-us**
Azure Terraform templates created. Template creates container groups that pull containers from the OpenLXP DockerHub organization. This does not bring up the actual app but demonstrates use of existing containers on Azure.

### CODE REVIEW

File: aws/aws_xia_network_setup.tf [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  availability_zone = element(data.aws_availability_zones.ecc-azs.names, 0)
+  vpc_id            = aws_vpc.ecc_vpc.id
+  cidr_block        = "10.0.1.0/24"
+
+  tags = {
+    Name = "xia_subnet1_useast1"
+  }
+}
+
+
```
Comment (mattdrayer) [inline aws/aws_xia_network_setup.tf:13]: Nit: looks like there's an extra empty line here

File: aws/aws_xss_network_setup.tf [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  availability_zone = element(data.aws_availability_zones.openlxp-azs.names, 0)
+  vpc_id            = aws_vpc.openlxp_vpc.id
+  cidr_block        = "10.0.14.0/24"
+
+  tags = {
+    Name = "xss_subnet_useast1"
+  }
+}
+
+
```
Comment (mattdrayer) [inline aws/aws_xss_network_setup.tf:13]: Nit: Extra line?  Or maybe we like the double-lines -- doesn't matter to me as long as we're consistent across scripts.

File: aws/task-definitions/xds_images.json [hunk 3]
```diff
@@ -0,0 +1,93 @@
+[
```
Comment (mattdrayer) [inline aws/task-definitions/xds_images.json:1]: Might be nice to include a comment at the top of all scripts describing their purpose, any key dependencies they have, etc.

**Review by mattdrayer — APPROVED** [review]
This PR looks fantastic -- great job with the overall script structure and the naming conventions.  Really just a couple nits and a general recommendation to add top-of-file comments script describing the purpose of each script and any key dependencies, assumptions, expectations, etc.

**Review by jdiouf — APPROVED** [review]

**Comment by micson-us** [comment]
closing this PR due to conflicts from updates

**Pull request closed without merging**